server/src/internalClusterTest/java/org/opensearch/search/aggregations/bucket/NestedIT.java

```diff
@@ -606,6 +608,7 @@ public void testNestedSameDocIdProcessedMultipleTime() throws Exception {
                     .endObject()
             )
             .get();
+        indexRandomForConcurrentSearch("idx4");
```

Comment:
should we call this after refresh rather?

Reply:
I don't think it makes a difference either way. Unless you have strong feelings about this I don't see a need to change it.

Reply:
I would prefer a refresh beforr indexing the bogus docs rather than two refresh calls one after the other

Reply:
I'm fine with making the change, but what is the difference you're concerned about here? The second refresh will be a no-op

Reply:
Changed

Reply:
Thanks for the changes, I am concerned if two refresh calls together could cause segment merge and impact the slice count number which is whole point why we are doing this.